veps/sig-compute/decentralized-migration/decentralized-migration-vep.md

```diff
@@ -0,0 +1,308 @@
+# VEP #Decentralized live migration
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
```

Comment:
nit: you can mark this as done.

nit2: can you add a link to the tracking issue in the PR description?

Reply:
I have updated the description to include a link to the tracking issue. For the first nit, I don't think I can do that until the VEP is merged, so I can then link the issue to the location. Maybe I am not understanding the process correctly.